Fronter/Resources/GeneralFAQ-READ.ME.FIRST.txt

```diff
@@ -0,0 +1,112 @@
+This FAQ is always current for all the converters. Please read it before you ask questions.
+
+NOTE: The converters do NOT support XBox/Gamepass installations, they cannot be used.
+
+========== GENERAL QUESTIONS AND OPERATION ============
+
+=== 1. Where do I download the latest converters from?
+
+From Github (https://github.com/ParadoxGameConverters/). Always Github.
+The Paradox forums (https://forum.paradoxplaza.com/forum/) and Steam Workshop mods we provide all point to Github.
+
+=== 2. How do I unpack the converter?
+
+Using any windows tool for unpacking .zip files. (Such as 7-Zip or WinRAR)
+
+=== 3. Where do I unpack the converter?
+
+Create a new folder on your Desktop or in your Documents Folder and unpack it there.
+Do *NOT* unpack it into program files, or into your game installation folders.
```

Comment:
If we advise creating a top directory for the converter files, we might as well do that in zip creation.
https://discord.com/channels/612683871112396800/612683871112396802/934793492868243556

Reply:
We do but one is not exclusive with the other. I have no issues telling them to unpack to Desktop\converter\eu4tovic2.